MacOSX/build-package.in

```diff
@@ -11,8 +11,7 @@ test -x ./configure || ./bootstrap
 BUILDPATH=${PWD}
 
 # Locate the latest OSX SDK
-SDKS_PATH="$(xcode-select -p)/Platforms/MacOSX.platform/Developer/SDKs"
-SDK_PATH="${SDK_PATH:-$SDKS_PATH/$(ls -1 ${SDKS_PATH} | sort -n -k2 -t. -r | head -1)}"
+SDK_PATH=$(xcrun --sdk macosx --show-sdk-path)
```

Comment:
Returns only active SDK
man xcrun

The active developer directory can be set using `xcode-select`, or via the
DEVELOPER_DIR environment variable. See the xcrun and xcode-select manual
pages for more information.